server/datastore/datastore.go

```diff
@@ -1,14 +1,23 @@
 package datastore
 
-import "time"
+import (
+	"time"
+	"github.com/jackc/pgx"
+)
 
+/* DB is an interface for the app database. Any struct used as the app database
+* must implement these functions */
 type DB interface {
-	Init() error
+	Init() error // Initialize pgx connection
+	Connection() *pgx.Conn // Fetch established pgx connection
```

Comment:
what does this function do? I'm confused. If it's to get the `pgx.Conn` instance it shouldn't live on the interface. Should just access the connection on the psql DB struct instead

Reply:
the reason this shouldn't live on the interface is because the interface isn't implementation specific. to return a `*pgx.Conn` implies that we'll always be using a postgres database but that's not necessarily the case (the in memory database is what we were gonna use in the meanwhile as we figured out the database part)

Reply:
Removed! We will test db interactions with See Long's code.